django_filters/filters.py

```diff
@@ -235,6 +240,12 @@ def filter(self, qs, value):
 
         return qs.distinct() if self.distinct else qs
 
+    def get_filter_predicate(self, v):
+        try:
+            return {self.name: getattr(v, self.field.to_field_name)}
+        except AttributeError:
+            return {self.name: v}
```

Comment:
Can't see how `hasattr` is faster... https://hg.python.org/cpython/file/default/Objects/object.c#l911 — seems to call `getattr` ???